pilot/pkg/model/service.go

```diff
@@ -143,9 +143,9 @@ const (
 	// ProtocolTCP declares the the port uses TCP.
 	// This is the default protocol for a service port.
 	ProtocolTCP Protocol = "TCP"
-	// ProtocolTCPTLS declares that the port carries TLS traffic on top of TCP
-	// TLS traffic is assumed to contain SNI as part of the handshake
-	ProtocolTCPTLS Protocol = "TCP_TLS"
+	// ProtocolTLS declares that the port carries TLS traffic.
+	// TLS traffic is assumed to contain SNI as part of the handshake.
+	ProtocolTLS Protocol = "TLS"
```

Comment:
Where is  this used ? Does it require config changes for users ? 
Is it upgrade safe and backward compat ?

Reply:
Its not used anywhere, i.e. its totally new and backward compatible